test/test.py

```diff
@@ -223,8 +223,6 @@ def testSZ3(self):
                              "Values should not be identical")
             self.assertTrue(numpy.allclose(original, output_data, atol=value),
                              "Values should be within tolerance")
-            self.assertTrue(numpy.alltrue(compressed == output_data),
```

Comment:
This is removed, other tests checks that both previously and newly compressed data respects the error bound from the original data.